keyboards/idyllic/pizzapad/keyboard.json

```
@@ -13,10 +13,21 @@
         "rows": ["GP6", "GP29", "GP28"]
     },
     "usb": {
-        "device_version": "0.0.1",
+        "device_version": "0.0.2",
         "pid": "0x5050",
         "vid": "0x7A79"
     },
+    "features": {
+        "mousekey": true,
```

While this is being updates, it should really be aligned to the general preferences for default config
```suggestion
        "bootmagic": true,
        "mousekey": true,
```